Patches/Vanilla Factions Expanded - Ancients/ThingDefs_Sentry.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Vanilla Factions Expanded - Ancients</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+
+		<!-- Remove refuelable property -->
+		<li Class="PatchOperationRemove">
+		<xpath>Defs/ThingDef[
+			defName = "VFEA_Turret_AncientSecurityTurret" or		
+			defName = "VFEA_Turret_AncientPointDefense"
+		]/comps/li[@Class = "CompProperties_Refuelable"]</xpath>
+		</li>
+
+		<!-- Replace vanilla thingClass -->
+		<li Class="PatchOperationReplace">
+      <xpath>Defs/ThingDef[
+			defName = "VFEA_Turret_AncientSecurityTurret" or		
+			@Name = "AncientPointDefenseTurretBase"	        		
+		]/thingClass</xpath>
+        <value>
+          <thingClass>CombatExtended.Building_TurretGunCE</thingClass>
+        </value>
+		</li>
+
+    <!-- Make turrets taller -->
+    <li Class="PatchOperationReplace">
+      <xpath>Defs/ThingDef[defName = "VFEA_Turret_AncientSecurityTurret"]/fillPercent</xpath>
```

Comment:
`@Name = "VFEA_Turret_AncientPointDefense" ` too or it has 50% fill otherwise

possibly also remove explosive comp to be consistent with other turrets in CE